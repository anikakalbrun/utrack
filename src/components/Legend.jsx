import React from 'react';
import {node} from 'prop-types';

Legend.propTypes = {
    children: node.isRequired
};

export default function Legend({children}) {
    return (
        <div className='legend'>{children}</div>
    );
}