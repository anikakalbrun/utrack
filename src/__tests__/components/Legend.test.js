import React from 'react';
import Legend from '../../components/Legend';

let wrapper;

describe('Legend', () => {

    it('renders correctly', () => {
        wrapper = renderer.create(<Legend children={<span>test</span>}/>).toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});
