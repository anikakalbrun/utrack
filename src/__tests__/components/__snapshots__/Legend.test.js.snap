// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Legend renders correctly 1`] = `
<div
  className="legend"
>
  <span>
    test
  </span>
</div>
`;
